index.js

```diff
@@ -11,6 +11,7 @@ const levelTranslate = new Map([
 , ['info', 'info']
 , ['http', 'debug']
 , ['verbose', 'debug']
+, ['debug', 'debug']  
```

Comment:
The linter is failing with trailing spaces on this line.  Please run `npm run test` prior to re-pushing anything.

Reply:
You can also run `npm run lint -- --fix` to auto-correct those issues